tests/unit/CorrectNameToNumber.spec.ts

```diff
@@ -49,8 +59,8 @@ describe.skip('CorrectNameToNumber', () => {
 
     // Verify data
     expect(nameToNumberInput.attributes('aria-checked')).toBe('false')
-    expect(wrapper.emitted('valid')).toBeUndefined()
-    expect(wrapper.emitted('update:nameRequest').legalName).toBe('Bobs Plumbing')
+    expect(wrapper.emitted('valid')).toStrictEqual([[true, true, false], [false, false, true]])
```

Comment:
Check other tests, where we "pop" the last emitted value, since that's the only one we really care about.

I want these tests to actually test that the code is working correctly (from a user point of view). I don't just want the tests to check for whatever happens to be the current value.